ai-creator/AIArtGenerator.js

```diff
@@ -0,0 +1,12 @@
+const deepai = require('deepai');
+
+deepai.setApiKey('quickstart-QUdJIGlzIGNvbWluZy4uLi4K') //This key free (untested)
```

Comment:
Can we make this key to be dynamically passed in? 
Can we also create our own account and use the newly generated API key?

Reply:
I will find out how to get our own key and I agree that we should pass this in dynamically.